matlab/schemas/+map/OptImageBar.m

```diff
@@ -239,34 +239,92 @@ function plotTight(obj,varargin)
             if isempty(keys); disp('Nothing found!'); return; end
             
             for ikey = 1:length(keys)
```

Comment:
Try `for key = keys'` instead of indexing with `ikey`

Reply:
I need key

Reply:
Not really but this is not a big deal